### ISSUE TITLE
Invent clever way to trigger notification when a first-timer finishes a pull request

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#3416]
We have a great set of first-timers-only issues at https://code.publiclab.org, and a welcomebot to thank people when they're successfully closed:

![image](https://user-images.githubusercontent.com/24359/45962859-e41d1400-bfef-11e8-91a0-dd9927d6433d.png)

(see https://github.com/publiclab/plots2/pull/3392#issuecomment-423772382) 

We can find closed pull requests where WelcomeBot has responded using this API query:

https://api.github.com/search/issues?o=desc&q=is%3Apr+is%3Aclosed+%22Congrats+on+merging+your+first+pull+request%21%22+user:publiclab&s=updated&type=Issues

But it would be great to trigger some kind of notification to be sent, by email/tweet/slack/chat, so we can be reminded to thank people afterwards! It's hard to keep track of all of the closed issues, so we could use some help with this one. 

Can you think of a clever integration to achieve this?

**Comment by gauravano** [comment]
So, it would be like a thank you message in form of Email or a tweet? People may not mention their twitter handle, so if we wanna be sure that they got our message, Emailing them from PL domain would be nice :)

**Comment by jywarren** [comment]
the WelcomeBot message now asks people for their twitter handle, so the
ideal is that this is triggered when they respond to /that/ prompt. But it
would still be VERY useful to just see all closed PRs that were that
person's first PR, which the above link shows via the API. I just can't
think of a simple way to trigger a message of any kind from that API
result... :-/

**Comment by Souravirus** [comment]
Hi @jywarren, can't we ask for their twitter handle when the first-timer open the pull request. So, that we get notified and then later thank them in twitter when the PR gets merged.

**Comment by jywarren** [comment]
ah, good idea! And good FTO -- have you used first-timers-bot yet?

https://github.com/hoodiehq/first-timers-bot

the file is under /.github/config.yml

**Comment by Souravirus** [comment]
Yeah I have used it while creating first-timers issues. I will make a fto issue.

**Comment by Souravirus** [comment]
@jywarren just wanted to ask if we want to add the message of Twitter handle on the first-timers-issue or the welcome-bot opening message.

**Comment by Souravirus** [comment]
In my opinion, this message should be on the welcome-bot opening message.

**Comment by jywarren** [comment]
that sounds great. Let's do it!

**Comment by Souravirus** [comment]
@jywarren I have made a new first-timers-issue #3485. Please see this. Just wanted to ask can we make a gci-candidate issue or leave it as a normal first-timers-issue.

**Comment by jywarren** [comment]
maybe best mark it as fto-candidate and gci-candidate for now... we have so
many newcomers it's getting a little tough to manage!

**Comment by Souravirus** [comment]
Ok I will
